task/harden-concourse-tasks/scripts/build.sh

```diff
@@ -157,6 +165,12 @@ ln -s /usr/local/go/bin/gofmt /usr/local/bin/gofmt
 
 go env -w GOBIN=/usr/local/bin
 
+pushd "/usr/local/bin"
+  go get github.com/Masterminds/goutils
```

Comment:
Might these need to be `go install` if you're trying to install them system-wide?

Reply:
Resolved this by updating project versions rather than trying to install system wide